### PR TITLE
Added string needed for change on #1766. 

### PR DESCRIPTION
**Pull request by maxxcrawford**
This does NOT solve #1766. Once localized broadly, we can update the referenced string.

### CODE REVIEW

File: locales/en/app.ftl [hunk 1]
```diff
@@ -12,6 +12,7 @@
 -brand-lockwise = Firefox Lockwise
 -brand-send = Firefox Send
 -brand-fpn = Firefox Private Network
+-brand-fpn-2 = Mozilla VPN
```
Comment (flodolo) [inline locales/en/app.ftl:15]: Why not `-brand-mozilla-vpn` at this point?

Comment (maxxcrawford) [reply locales/en/app.ftl:15]: Good call. That'll fix any confusions later too. Updating!

**Comment by groovecoder** [comment]
@flodolo - are you saying we should include the updates to the promo CTA where this new brand name will be used?

**Comment by flodolo** [comment]
> @flodolo - are you saying we should include the updates to the promo CTA where this new brand name will be used?

I think so, there's not much value in exposing the new term alone, when it's not used anywhere.

**Comment by lesleyjanenorton** [comment]
@maxxcrawford do you want to update `-brand-fpn-2` to `-brand-mozilla-vpn` or should we close this in favor of #1771 ?

**Comment by lesleyjanenorton** [comment]
Closing in favor of #1771

**Pull request closed without merging**